src/main/java/uk/gov/justice/digital/delius/data/api/Conviction.java

```diff
@@ -23,4 +23,10 @@
     private Sentence sentence;
     private KeyValue latestCourtAppearanceOutcome;
     private Custody custody;
+    private Court responsibleCourt;
```

Comment:
maybe take the opportunity to add a couple of Swagger API annotations on these two new fields and the latestCourtAppearanceOutcome field.

Sadly they have always been missing but given it isn't clear the difference it would be nice to add them.